ssg/build_yaml.py

```diff
@@ -1441,6 +1442,8 @@ def _load_group_process_and_recurse(self, guide_directory):
         if self.loaded_group:
             if self.parent_group:
                 self.parent_group.add_group(self.loaded_group)
+                if hasattr(self.parent_group, "platforms") and self.parent_group.platforms:
+                    self.loaded_group.inherited_platforms.update(self.parent_group.platforms)
```

Comment:
The `add_group()` method from `Group` class calls a method named `_pass_our_properties_on_to()`, which passes down to its child group or rule its `platform`.